app/views/miq_policy/_policy_list.html.haml

```diff
@@ -1,10 +1,13 @@
 #policy_list_div
   - if x_active_tree == :policy_tree && @view
     - if @view.table.data.empty?
+      - mode, model = @sb[:folder].split('-')
```

Comment:
This was funny.  `@mode` is never set at all.  `@sb[:folder]` has the form `"compliance-extManagementSystem"`, which ui_lookup falled onto titleizing to "Compliance Ext Management System".
It was missed because "No Compliance Host Policies" is close enough to "No Host Compliance Policies"...